packages/transaction-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/transaction-controller",
-  "version": "21.2.0",
```

Comment:
I think you had mentioned this in Slack, but due to the breaking change, we should release a major version.

``` suggestion
  "version": "22.0.0",
```

Reply:
Done in https://github.com/MetaMask/core/pull/3915/commits/78d9aea07deaed854110580a9200c9bcecce7213